File: ex2/main.c
#include <stdio.h>

int main() {
    double fahrenheit, celsius;
    printf("enter degree in fahrenheit: ");
    scanf("%lf", &fahrenheit);
    celsius = (5.0 / 9.0) * (fahrenheit - 32);
    printf("%.2lf fahrenheit = %.2lf celsius\n", fahrenheit, celsius);
    return 0;
}

File: ex2/main.go
package main

import (
    "fmt"
)

func main() {
    var fahrenheit, celsius float64
    fmt.Print("enter degree in fahrenheit: ")
    fmt.Scanf("%f", &fahrenheit)
    celsius = (5.0 / 9.0) * (fahrenheit - 32)
    fmt.Printf("%.2f fahrenheit = %.2f celsius\n", fahrenheit, celsius)
}
